### ISSUE TITLE
An error occurred while parsing the item

### ISSUE THREAD

**Issue by Devalock** [SnosMe/awakened-poe-trade#1426]
I've met the same problem in https://github.com/SnosMe/awakened-poe-trade/issues/1336.
Any ideas? Thanks!
![微信图片_20240804171818](https://github.com/user-attachments/assets/dbab674e-7571-494e-b811-43be68e8c4dd)

**Comment by SnosMe** [comment]
Can you paste this item text here please

**Comment by Devalock** [comment]
> Can you paste this item text here please

Sure, I randomly choose another equipment though
![image](https://github.com/user-attachments/assets/1be7cb6c-315d-4542-bf08-2e4df532e41e)


Item Class: Gloves
Rarity: Rare
Gloom Paw
Silk Gloves
--------
Energy Shield: 20
--------
Requirements:
Level: 50
Str: 49
Dex: 62
Int: 78
--------
Sockets: B-G-B-B 
--------
Item Level: 27
--------
{ Prefix Modifier "Essences" — Life }
+6(5-14) to maximum Life
{ Prefix Modifier "Glinting" (Tier: 4) — Damage, Physical, Attack }
Adds 1 to 2 Physical Damage to Attacks
{ Suffix Modifier "of the Lynx" (Tier: 9) — Attribute }
+15(13-17) to Dexterity
{ Suffix Modifier "of the Salamander" (Tier: 7) — Elemental, Fire, Resistance }
+15(12-17)% to Fire Resistance
{ Suffix Modifier "of Rejuvenation" (Tier: 1) — Life, Attack }
Gain 2 Life per Enemy Hit with Attacks

**Comment by SnosMe** [comment]
It doesn't error for me, so probably the only reasone I have on mind is you have some ancient version of the app

**Comment by Devalock** [comment]
> It doesn't error for me, so probably the only reasone I have on mind is you have some ancient version of the app

OK, I'll try reboot and reinstall the app

**Comment by Devalock** [comment]
> > It doesn't error for me, so probably the only reasone I have on mind is you have some ancient version of the app
> 
> OK, I'll try reboot and reinstall the app

Nah, it still doesn't work. However the price check function of POE overlay seems working and I can visit poe trade site, so it shouldn't be network problem or something

**Comment by streamsteel** [comment]
> > > It doesn't error for me, so probably the only reasone I have on mind is you have some ancient version of the app
> > 
> > 
> > OK, I'll try reboot and reinstall the app
> 
> Nah, it still doesn't work. However the price check function of POE overlay seems working and I can visit poe trade site, so it shouldn't be network problem or something

Okie, I think I found the reason.
Before I used Chinese plug-in, it will set game to French, and I changed game language to English and deleted the Awakend Plugin config folder, and it works.

**Comment by Devalock** [comment]
> > > > It doesn't error for me, so probably the only reasone I have on mind is you have some ancient version of the app
> > > 
> > > 
> > > OK, I'll try reboot and reinstall the app
> > 
> > 
> > Nah, it still doesn't work. However the price check function of POE overlay seems working and I can visit poe trade site, so it shouldn't be network problem or something
> 
> Okie, I think I found the reason. Before I used Chinese plug-in, it will set game to French, and I changed game language to English and deleted the Awakend Plugin config folder, and it works.

Excellent, it works!
而你，我的朋友，是真正的英雄！

**Comment by Coetley** [comment]
Hello,

Does the issue is solved only if the game is in English ? Because when going back to French, it doesn't work again.

**Comment by sadwqq555** [comment]
> > > > 这对我来说没有错误，所以可能我想到的唯一原因是您有一些古老版本的应用程序
> > > 
> > > 
> > > 好的，我会尝试重新启动并重新安装该应用程序
> > 
> > 
> > 不，它仍然不起作用。但是，POE覆盖的价格检查功能似乎可以正常工作，我可以访问poe交易网站，所以应该不是网络问题或其他问题
> 
> Okie，我想我找到了原因。在我使用中文插件之前，它会将游戏设置为法语，我将游戏语言更改为英语并删除了 Awakend 插件配置文件夹，它可以正常工作。

那么中文化还能用吗，只能一直英文吗

**Comment by Devalock** [comment]
> > > > > 这对我来说没有错误，所以可能我想到的唯一原因是您有一些古老版本的应用程序
> > > > 
> > > > 
> > > > 好的，我会尝试重新启动并重新安装该应用程序
> > > 
> > > 
> > > 不，它仍然不起作用。但是，POE覆盖的价格检查功能似乎可以正常工作，我可以访问poe交易网站，所以应该不是网络问题或其他问题
> > 
> > 
> > Okie，我想我找到了原因。在我使用中文插件之前，它会将游戏设置为法语，我将游戏语言更改为英语并删除了 Awakend 插件配置文件夹，它可以正常工作。
> 
> 那么中文化还能用吗，只能一直英文吗

Awakened poe trade有一个中文版（好像是别人开发的），是可以与中文汉化联用的。使用某个特定的汉化方法时（A大汉化）时游戏里设置的语言实际上是法语，所以我猜查价应该实际上是基于法语的，这个失效的问题是因为使用了中文版后config文件被改变了，这时候再不使用汉化而使用英语进行游戏，就有语言不对齐的问题（查价基于法语而游戏使用英语）。总之，你要用汉化就用对应的中文版Awakened，用英文就删除config并使用原装的英文版Awakened。

Awakened PoE Trade has a Chinese version (which seems to be developed by someone else) that can be used in conjunction with the Chinese localization. When using a specific localization method, the language set in the game is actually French. So I suspect that the price checking is actually based on French. The issue above arises because after using the Chinese version, the config file is altered. If you then switch back to playing in English without using the localization, there’s a language mismatch (price checking is based on French, but the game is in English). In short, if you want to use the localization, use the corresponding Chinese version of Awakened. If you prefer English, delete the config and use the original English version of Awakened.
